h2o/README.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+title: "tidymodels and h2o integration"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+## Grid search differences
+
+There are some differences between tidymodels grid tuning and h2o:
+
+* `h2o.grid()` seems to work on one resample at a time.
```

Comment:
Not sure if this is helpful to note, but we have a `parallelism` arg for `h2o.grid()` which allows for multiple models to be trained in parallel.  You specify an integer for how many models to train at once, e.g. `parallelism = 5`.  So in theory it can work with multiple resamples at a time.